force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -271,11 +347,16 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer findWordInList(String wordToFind, List<String> words) {
         // The variable to store the index of the word
+        Integer index = -1;
 
         // Loop through the list of words
-            // If the current word is the word to find, exit the loop
-
-        return null; // Replace null with the variable you used to store the result
+        for(Integer i=0; i< words.size(); i++){
+        // If the current word is the word to find, exit the loop
+            if(wordToFind == words.get(i)){
+                index = i;
+            }
+        }
+        return index; // Replace null with the variable you used to store the result
     }
```

Comment:
In your loop, you want to break out the second you find a match. Imagine you have a collection with 1,000,000 members. And you find your match at index 0. You don't want to waste your resources checking the rest of the list. Exit out and return the index. 
```
 public static Integer findWordInList(String wordToFind, List<String> words) {
     for(Integer i = 0; i < words.size(); i++){
         if(words[i] == wordToFind){
             return i;
         }
     }
        return -1;
    }
 ```